src/backend/gl/src/info.rs

```diff
@@ -469,6 +471,11 @@ pub(crate) fn query_all(gl: &GlContainer) -> (Info, Features, LegacyFeatures, Li
         legacy |= LegacyFeatures::INSTANCED_ATTRIBUTE_BINDING;
     }
 
+    let per_draw_buffer_blending = info.is_supported(&[Core(4, 0), Es(3, 2)]) && !info.is_webgl();
```

Comment:
should add "EXT_draw_buffers2" extension here

Reply:
Good to know, will do that!